osde2e/osde2e-wrapper.py

```diff
@@ -270,6 +270,9 @@ def main():
     if not args.es_index_only and not args.account_config:
         parser.error('the following arguments are required: --account-config')
 
+    if args.only_delete_clusters and not args.path:
+        parser.error('the following arguments are required: --path')
```

Comment:
Can we update this to say "the following arguments are required when specifying --only-delete-clusters"? Its a little more obvious for the user then. We should do the same with the if above this as well.

Reply:
ok, I will change it, but consider that it will be only shown when --only-delete-clusters parameter is used.